shell/platform/android/io/flutter/view/AccessibilityViewEmbedder.java

```diff
@@ -195,6 +195,7 @@ private void cacheVirtualIdMappings(@NonNull View embeddedView, int originId, in
         flutterIdToOrigin.put(flutterId, origin);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
@mklim do you happen to know if we can just stop setting getBoundsInParent given that we're always setting getBoundsInScreen which is supported starting API 14 ?

Reply:
That sounds right to me. The deprecation warning on `getBoundsInParent` says to use `getBoundsInScreen` instead, and it was added API 14.

Reply:
ok - in that case I'll remove the copying of the BoundsInParent from the origin node